src/reference/asciidoc/jms.adoc

```diff
@@ -397,6 +397,23 @@ __NOT ALLOWED__
 User-supplied correlation ids are not permitted with a reply listener; the gateway will not initialize with this
 configuration.
 
+[[jms-async-gateway]]
+==== Async Gateway
+
+Starting with _version 4.3_, you can now specify `async="true"` (or `setAsync(true)`) when configuring the outbound
```

Comment:
s/setAsync/setAsyncReplySupported

Or maybe it still isn't late to rename that target property (and its getter/setter) just to the `async` and  that's all?